File: 2015/02/main.py
# read file

file = open('./input.txt', 'r')
input = file.read()

# go line by line
each_line = input.split('\n')
each_line.pop()

x = each_line[0].split('x')
sa = 0
for i in range(len(each_line)):
    [l, w, h] = each_line[i].split('x')
    [l, w, h] = [int(l), int(w), int(h)]
    lw = 2 * l * w
    wh = 2 * w * h
    hl = 2 * h * l
    small = min(lw, wh, hl) / 2
    sa += lw
    sa += wh
    sa += hl
    sa += small
    print(sa)

print(sa)

#for i in range(len(x)):
#    x[i] = int(x[i])

File: 2015/02/main.rs
use std::fs::File;
use std::io::Read;

fn main() {
    // try to open file
    let mut file: File = match File::open("input.txt") {
        Ok(file) => file,
        Err(error) => panic!("Problem: {}", error),
    };

    // read it to string
    let mut input: String = String::new();
    input = match file.read_to_string(&mut input) {
        Ok(_) => input,
        Err(error) => panic!("Problem: {}", error),
    };

    //delete
    println!("{}", input);
}
